app/indices/account_index.rb

```diff
@@ -15,7 +15,7 @@
   indexes users.id,                            as: :user_id
 
   set_property field_weights: { name: 2 }
-
+  set_property charset_table: "0..9, A..Z->a..z, a..z, U+23, U+25, U+27, U+2A..U+2C, U+2E, U+3A, U+3B, U+3F, U+5F, U+60, U+7B, U+7D"
```

Comment:
They  are alredy defined in https://github.com/3scale/porta/blob/91c63772aab5068952293011060f142a03e30635/config/thinking_sphinx.yml#L17 and https://github.com/3scale/porta/blob/91c63772aab5068952293011060f142a03e30635/config/docker/thinking_sphinx.yml#L17

Please modify them there

Reply:
@hallelujah - @kevprice83 and I thought it would have a smaller impact to restrict this change to the charset in the account index only by editing `account_index.rb`, isn't that how it works? I'll wait for your confirmation before applying the changes to `thinking_sphinx.yml` instead. Thanks!

Reply:
@samu91gn @kevprice83 Yes that is better indeed!